spec/lib/bgs/vnp_veteran_spec.rb

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe BGS::VnpVeteran do
+  let(:user_object) { FactoryBot.create(:evss_user, :loa3) }
+  let(:user_hash) do
+    {
+      participant_id: user_object.participant_id,
+      ssn: user_object.ssn,
+      first_name: user_object.first_name,
+      last_name: user_object.last_name,
+      external_key: user_object.common_name || user_object.email,
+      icn: user_object.icn
+    }
+  end
+  let(:all_flows_payload) { FactoryBot.build(:form_686c_674) }
+  let(:current_marriage_details) do
+    {
+      'current_marriage_details' => {
+        'date_of_marriage' => '2014-03-04',
+        'location_of_marriage' => {
+          'state' => 'California',
+          'city' => 'Slawson'
+        },
+        'marriage_type' => 'OTHER',
+        'marriage_type_other' => 'Some Other type',
+        'view:marriageTypeInformation' => {}
+      }
+    }
+  end
+  let(:formatted_payload) do # This is here for the mocks since they receive formatted params
```

Comment:
What are "the mocks" in this context? As someone without context, I'm not entirely sure what this comment is saying. Could you expand a little on this comment?

Reply:
It's for some rspec/test mocks, you're right it is confusing, I had to read the code to figure out what I meant. After thinking about it, I decided to remove the comment since it caused us more confusion than help.